### DISCUSSION TITLE
V2.0.1 - lift, selection & color

### DISCUSSION THREAD

**Discussion by brentbatch** [brentbatch/Advanced-Blueprint-Tool-V2#6] [General]
changes:
- edit title/description of a blueprint:

https://user-images.githubusercontent.com/38281597/131229737-7d40b1b5-f105-4d3f-9ef6-f879991c3a6a.mp4

- you can now open the blueprint folder / workshop page

https://user-images.githubusercontent.com/38281597/131229534-9bc62897-8ddc-4785-ac79-b83fbf2c80bc.mp4

- you can now delete blueprints

https://user-images.githubusercontent.com/38281597/131229589-9adf1db0-3a60-4f31-983f-4c14f8a509a4.mp4

- multiselection in the direction of your new selection when holding shift

https://user-images.githubusercontent.com/38281597/131229601-1ee34a9f-dfca-4fb3-8303-314f7ef9b1f5.mp4

- previewing/applying a color & undo color (only for single part selection)

https://user-images.githubusercontent.com/38281597/131229644-6f91334d-9c0b-4516-8a06-49908adcae36.mp4

- saving/picking and using a color

https://user-images.githubusercontent.com/38281597/131229613-4cce0d40-65bf-4a09-8590-5576b9f8702b.mp4

- applying color to a bigger selection:

https://user-images.githubusercontent.com/38281597/131229655-59ccb3c1-ebf2-465e-ac46-2700856d3c6a.mp4

controls:
> move: WASD
  sprint: SHIFT
  up: SPACE
  down: CTRL
  interact: E LMB RMB
  toggle hotbar: MMB (middle mouse button click)
  scroll through hotbar: MMB (scroll)
  focus on screen: click on the background/screen
  toggle focus: C
  escape: ESC

tools:
>  lift: load/save & clear scene features. 'e' to open lift menu, searching works, loading works, saving whatever is in the scene to the selected blueprint works (only overwrite, no 'save as' yet)
  mutator: move feature works. select block(s) with left click + drag for a bigger selection. the selection box can be resized by selecting a side and dragging it out. the selected block(s) can be moved using the visible gizmo. single click selection selects a single part even though multiple might be clipping. right click to undo selection. the rotate function can be centered on grid center or corner using 'a', the center can be moved by dragging the selection gizmo.

Please mention any and all bugs/issues you may find.

<hr /><em>This discussion was created from the release <a href='https://github.com/brentbatch/Advanced-Blueprint-Tool-V2/releases/tag/v2.0.1'>V2.0.1 - lift, selection & color</a>.</em>

**Comment by jeffthejeffthejeff** [comment]
are you thinking of porting to mac?

**Comment by sanchos228pro** [comment]
it returns an error when saving :

Something went wrong while saving
your creation to this blueprint!
Check the log for more information
